lib/src/components/NxCollapsibleItemsSelect/commonTypes.ts

```diff
@@ -34,7 +35,10 @@ export interface CommonProps<T extends Option = Option> {
 
 export const optionPropType = PropTypes.shape({
   id: requiredNullableString,
-  name: PropTypes.string.isRequired
+  name: PropTypes.oneOfType([
+    PropTypes.string.isRequired,
+    PropTypes.node.isRequired
+  ]).isRequired
```

Comment:
Pretty sure node includes string
```suggestion
  name: PropTypes.node.isRequired
```